dataset-spec/template.schema.json

```diff
@@ -142,88 +204,123 @@
-                                 "examples": [
-                                     ["viridis_r", "turbo", "prism", "Spectral"]
-                                 ]
-                             },
-                             "color_value_min": {
-                                 "type": "number",
-                                 "description": "minimum value for colour scale",
-                                 "examples": [-10500, 0, 1.735e-9, 4.5e6]
-                             },
-                             "color_value_max": {
-                                 "type": "number",
-                                 "description": "maximum value for colour scale",
-                                 "examples": [-9500, 0, 2.978e-9, 8.5e6]
-                             },
-                             "metadata": {
-                                 "type": "object"
+                            }
+                        }
+                    }
+                },
+                "sources": {
+                    "type": "array",
+                    "description": "Array of data source definitions",
+                    "items": {
+                        "type": "object",
+                        "additionalProperties": false,
+                        "required": [
+                            "name",
+                            "variable_names",
+                            "store_id"
+                        ],
+                        "dependentRequired": {
+                            "variable_names": ["store_id"],
+                            "store_id": ["variable_names"],
+                            "variables": ["download_url"],
+                            "download_url": ["variables"]
+                        },
+                        "properties": {
+                            "name": {
+                                "type": "string",
+                                "description": "human-readable title of this data source",
+                                "examples": ["Sentinel Hub S2L2A"]
+                            },
+                            "variable_names": {
+                                "type": "array",
+                                "description": "array of variable names provided by this data store",
+                                "examples": [["B06", "SCL"]]
+                            },
+                            "store_id": {
+                                "type": "string",
+                                "description": "xcube data store identifier",
+                                "examples": ["sentinelhub"]
+                            },
+                            "store_params": {
+                                "type": "object",
+                                "description": "xcube data store parameters",
+                                "examples": [{"num_retries": 80}]
+                            },
+                            "variable_open_params": {
```

Comment:
Should be `dataset_open_params` or `open_data_params`, pls refer into xcube data store framework.

Reply:
Updated to `data_params` as discussed.